presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -72,29 +71,6 @@ public void testInsert()
         // TODO Support these test once kudu connector can create tables with default partitions
     }
 
-    @Override
-    public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
-    {
-        // TODO Support these test once kudu connector can create tables with default partitions
-    }
-
-    @Test
-    @Deprecated // TODO remove when testDataMappingSmokeTest is enabled for Kudu
-    public void testKuduPredicatePushdown()
-    {
-        assertUpdate("CREATE TABLE IF NOT EXISTS test_is_null (" +
-                "id INT WITH (primary_key=true), " +
-                "col_nullable bigint with (nullable=true)" +
-                ") WITH (" +
-                " partition_by_hash_columns = ARRAY['id'], " +
-                " partition_by_hash_buckets = 2" +
-                ")");
-
-        assertUpdate("INSERT INTO test_is_null VALUES (1, 1)", 1);
-        assertUpdate("INSERT INTO test_is_null(id) VALUES (2)", 1);
-        assertQuery("SELECT id FROM test_is_null WHERE col_nullable = 1 OR col_nullable IS NULL", "VALUES (1), (2)");
```

Comment:
This is now covered by `testDataMappingSmokeTest`
https://github.com/prestosql/presto/runs/581310253?check_suite_focus=true